.github/workflows/ci-zkVMs.yml

```diff
@@ -0,0 +1,33 @@
+name: CI - All zkVMs
+
+on:
+  workflow_call:
+  push:
+    branches: ["main"]
+    paths:
+      - "pipelines/**"
+      - "harness/**"
+  pull_request:
+    paths:
+      - "pipelines/**"
+      - "harness/**"
+env:
+  CARGO_TERM_COLOR: always
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-test-native:
+    uses: ./.github/workflows/ci-native.yml
+
+  build-test-risc0:
+    uses: ./.github/workflows/ci-risc0.yml
+
+  build-test-sp1:
+    uses: ./.github/workflows/ci-sp1.yml
+
+  build-test-sgx-all:
```

Comment:
sgx isnt a zkVM so hmm. do we want to run this here @Brechtpd ?

if not, we can remove it @kimo-ice, if so, maybe we can rename it to ci-all.

Reply:
misleading name and contents, sgx contents should not exist if it's called ci-zkVM I think.

Reply:
should just be called All provers, ci -provers.yml I guess